units/src/amount/mod.rs

```diff
@@ -67,16 +67,17 @@ pub use self::{
 /// ```
 #[derive(Debug, Clone, Copy, Eq, PartialEq, Hash)]
 #[non_exhaustive]
+#[allow(clippy::doc_markdown)]
```

Comment:
nit

```suggestion
#[allow(clippy::doc_markdown)] // Clippy thinks `µBTC` needs backticks.
```

FYI, in this repo 'nit' means 'just change this either if you want to or if you are rebasing anyways, its not super important. Or do a follow up PR if you can be bothered'

Definitely don't force push this one because it has 3 acks already - force pushing makes everyone have to re-review.

In general, any `allow` is better with a code comment.